lib/gadget/gadget.vala

```diff
@@ -70,6 +73,27 @@ namespace Frida.Gadget {
 		mutex.unlock ();
 	}
 
+	private bool try_get_listen_address (out string listen_address, out uint16 listen_port, out string listen_uri) {
```

Comment:
This method is lower level than `start()`, so it should be below it. Perhaps right below it, so it's easy to visually spot all the environment variables being accessed.